### ISSUE TITLE
Improve documentation for accessibility 

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#15332]
Accessibility main page: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/accessibility

## Issue Description
_What details are necessary for understanding the specific work or request tracked by this issue?_

---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by rroueche** [comment]
Idea for a content flow that has worked nicely for the va mobile app team: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/testing/Accessibility%20Test%20Plan.md

**Comment by rroueche** [comment]
Also feedback from Ryan T about starting to include WCAG 2.1 specs in our "how to do accessibility" content for testing and categorization of defects.

"we know 508 standard will soon require adherence to WCAG 2.1 specifications, which is focused on mobile and cognitive issues. Is it in our better interest to start incorporating 2.1 into what we’re building now (future-proofing-ish)? When login.gov developed its proofing requirements, we followed the NIST spec which had not been “officially required” but was released (meaning, we knew what the spec was, but it was not yet required to adopt it), and it that strategy definitely worked in our favor— @John Rahaghi can probably speak to this if you had any questions. Anyway, I think it’s something we should consider and would love to hear anyone’s thoughts."
